docker/create-pr.sh

```diff
@@ -27,7 +27,7 @@ for ENV in $(echo $ENVS | tr "," " "); do
       git checkout -b ${BRANCH}
 
       cd "${ENV}"
-      for APP in ckan pycsw solr; do
+      for APP in ckan pycsw; do
```

Comment:
```suggestion
      for APP in ckan; do
```

Reply:
Removed pycsw from create-pr.sh, will create another script to update pycsw and solr to latest images